### PR TITLE
BD-2977 Create reference page for Query Builder templates

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-2977**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/query_builder/query_templates.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+| Email performance by country | For each country, you'll see the following metrics: sends, indirect open rate, and direct open rate. Country is the country of the user at the time of push send. <br><br> ![]({% image_buster /assets/img_archive/query_builder_q3.png %}) |
+| Email Subscription Change Logs | For each user, you'll see the metrics that were logged about their subscription change, such as their email address, subscription status, the time their status was changed, and the associated Canvas or campaign. |
+| Email subscription group opt-ins and opt-outs | For each week, you'll see the number of unique user opt-ins and opt-outs of any email subscription group. s<br><br> ![]({% image_buster /assets/img_archive/query_builder_q2.png %}){: style="max-width:70%;"} |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/query_builder/query_templates.md:61]:
```suggestion
| Email subscription group opt-ins and opt-outs | For each week, you'll see the number of unique user opt-ins and opt-outs of any email subscription group. <br><br> ![]({% image_buster /assets/img_archive/query_builder_q2.png %}){: style="max-width:70%;"} |
```

File: _docs/_user_guide/data_and_analytics/query_builder/query_templates.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+nav_title: Query Templates
+article_title: Query Builder Templates
+page_order: 0
+page_type: reference
+toc_headers: h2
+description: "This reference article lists the types of reports you can create using Braze data from Snowflake in the Query Builder."
+tool: Reports
+---
+
+# Query Builder Templates
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/query_builder/query_templates.md:11]:
```suggestion
# Query Builder templates
```

File: _docs/_user_guide/data_and_analytics/query_builder/query_templates.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+| Purchases and revenue for variants or steps, by segment | This report provides metrics for the variants or Canvas steps of the messages sent to each segment. <br><br> Purchase metrics are unique over the entirety of the reporting period. One user can generate at most one purchase. Revenue takes into account every purchase from the reporting period. |
+| Top/bottom messaging for purchases | This report shows purchase metrics for the top or bottom campaigns, Canvases, or Canvas steps. Each row is a campaign, Canvas, or Canvas step. You must specify whether to display the top or bottom performers, and the specific metric to run this analysis for (such as *Unique purchases upon receipt*, *Revenue upon receipt*, *Unique recipients*). <br><br> The rows in top performer reports will be ordered from best to worst, while the rows in bottom performer reports will be ordered from worst to best. |
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/query_builder/query_templates.md:34]:
```suggestion
| Channel engagement and revenue | For each channel, you'll see all engagement metrics for that channel (for example, opens and clicks), revenue, number of transactions, and average price. {::nomarkdown} <ul> <li> <i>Number of transactions:</i> Number of purchase events </li> <li> <i>Average price:</i> Revenue divided by transactions </li> </ul> {:/} ![]({% image_buster /assets/img_archive/channel_engagement_revenue.png %}) |
| Purchases and revenue by segment | This report provides metrics for the messages sent for a specific segment. <br><br> Purchase metrics are unique throughout the reporting period. One user can generate at most one purchase. Revenue takes into account every purchase from the reporting period. |
| Purchases and revenue for variants or steps, by segment | This report provides metrics for the variants or Canvas steps of the messages sent to each segment. <br><br> Purchase metrics are unique throughout the reporting period. One user can generate at most one purchase. Revenue takes into account every purchase from the reporting period. |
| Top/bottom messaging for purchases | This report shows purchase metrics for the top or bottom campaigns, Canvases, or Canvas steps. Each row is a campaign, Canvas, or Canvas step. You must specify whether to display the top or bottom performers, and the specific metric to run this analysis for (such as *Unique purchases upon receipt*, *Revenue upon receipt*, *Unique recipients*). <br><br> The rows in top performer reports will be ordered from best to worst, while the rows in bottom performer reports will be ordered from worst to best. |
```

File: _docs/_user_guide/data_and_analytics/query_builder/query_templates.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+| Top/bottom messaging for email engagement | This report shows email engagment metrics for the top or bottom campaigns, Canvases, or Canvas steps. You must specify whether to display the top or bottom performers, and the specific metric to run this analysis for (such as *Sent*, *Soft Bounces*, and *Unique Opens*). <br><br> The rows in top performer reports will be ordered from best to worst, while the rows in bottom performer reports will be ordered from worst to best. <br><br> ![]({% image_buster /assets/img_archive/top-bottom-email.png %}) |
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/query_builder/query_templates.md:63]:
```suggestion
| Email bounces per domain | The number of bounces per email domain, broken down into total bounces, hard bounces, and soft bounces. <br> ![]({% image_buster /assets/img_archive/query_builder_q4.png %}){: style="max-width:60%;"} |
| Email delivery metrics by day | This report provides metrics for the messages sent on each day, such as how many emails were sent, delivered, soft bounced, and hard bounced. <br><br> All metrics are unique throughout the reporting period. For example, if a welcome email soft bounced one time on November 21, two times on November 22, and was never delivered: {::nomarkdown} <ul><li> The <i>Soft Bounces</i> metric for November 21 increases by one.</li><li> The <i>Soft Bounces</i> metric for November 22 is not affected. </li></ul>{:/} ![]({% image_buster /assets/img_archive/email_delivery_day.png %})|
|  Email engagement metrics by segment | This report provides metrics for the messages sent to each segment, such as how many emails were sent, delivered, soft bounced, and hard bounced. <br><br> All metrics are unique throughout the reporting period. For example, if a welcome email soft bounced one time on November 21, two times on November 22, and was never delivered: {::nomarkdown} <ul><li> The <i>Soft Bounces</i> metric for November 21 increases by one. </li><li> The <i>Soft Bounces</i> metric for November 22 is not affected.</li></ul>{:/} ![]({% image_buster /assets/img_archive/email_engagement_segment.png %}) |
| Email engagement metrics for variants or steps, by segment | This report provides metrics for the variants or Canvas steps of the messages sent to each segment. These metrics include how many emails were sent, delivered, soft bounced, and hard bounced. <br><br> All metrics are unique throughout the reporting period. For example, if a welcome email soft bounced one time on Nov 21, two times on Nov 22, and was never delivered: {::nomarkdown} <ul><li> The <i>Soft Bounces</i> metric for Nov 21 increases by one. </li> <li> The <i>Soft Bounces</i> metric for Nov 22 is not affected.</li></ul> {:/} |
| Email performance by country | For each country, you'll see the following metrics: sends, indirect open rate, and direct open rate. Country is the country of the user at the time of push send. <br><br> ![]({% image_buster /assets/img_archive/query_builder_q3.png %}) |
| Email Subscription Change Logs | For each user, you'll see the metrics that were logged about their subscription change, such as their email address, subscription status, the time their status was changed, and the associated Canvas or campaign. |
| Email subscription group opt-ins and opt-outs | For each week, you'll see the number of unique user opt-ins and opt-outs of any email subscription group. <br><br> ![]({% image_buster /assets/img_archive/query_builder_q2.png %}){: style="max-width:70%;"} |
| Email URLs clicked | This report shows the number of clicks each link in an email had. To run this report, you'll need to specify the API identifier for a campaign or Canvas. You can find a campaign's API identifier at the bottom of that campaign's details page and the Canvas API identifier under **Analyze Variants**. <br><br> For each de-personalized link, you'll see a count of clicks. Your CSV download will include the user IDs of all users that clicked, the link they clicked on, and a timestamp of when they clicked. <br><br> *De-personalized URLs:* URLs that are stripped of Liquid tags. <br><br> ![]({% image_buster /assets/img_archive/query_builder_q5.png %}){: style="max-width:70%;"} |
| Top/bottom messaging for email engagement | This report shows email engagement metrics for the top or bottom campaigns, Canvases, or Canvas steps. You must specify whether to display the top or bottom performers, and the specific metric to run this analysis for (such as *Sent*, *Soft Bounces*, and *Unique Opens*). <br><br> The rows in top performer reports will be ordered from best to worst, while the rows in bottom performer reports will be ordered from worst to best. <br><br> ![]({% image_buster /assets/img_archive/top-bottom-email.png %}) |
```

**Review by lydia-xie — APPROVED** [review]
lgtm left some suggestions for your review!

File: _docs/_user_guide/data_and_analytics/query_builder/query_templates.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+| Email engagement metrics for variants or steps, by segment | This report provides metrics for the variants or Canvas steps of the messages sent to each segment. These metrics include how many emails were sent, delivered, soft bounced, and hard bounced. <br><br> All metrics are unique throughout the reporting period. For example, if a welcome email soft bounced one time on Nov 21, two times on Nov 22, and was never delivered: {::nomarkdown} <ul><li> The <i>Soft Bounces</i> metric for Nov 21 increases by one. </li> <li> The <i>Soft Bounces</i> metric for Nov 22 is not affected.</li></ul> {:/} |
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/query_builder/query_templates.md:58]:
```suggestion
| Email engagement metrics for variants or steps, by segment | This report provides metrics for the variants or Canvas steps of the messages sent to each segment. These metrics include how many emails were sent, delivered, soft bounced, and hard bounced. <br><br> All metrics are unique throughout the reporting period. For example, if a welcome email soft bounced one time on November 21, two times on November 22, and was never delivered: {::nomarkdown} <ul><li> The <i>Soft Bounces</i> metric for November 21 increases by one. </li> <li> The <i>Soft Bounces</i> metric for November 22 is not affected.</li></ul> {:/} |
```

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**